packages/forms/src/directives/reactive_directives/form_control_name.ts

```diff
@@ -238,7 +242,9 @@ export class FormControlName extends NgControl implements OnChanges, OnDestroy {
    * Returns an array that represents the path from the top-level form to this control.
    * Each index is the string name of the control on that level.
    */
-  get path(): string[] { return controlPath(this.name, this._parent !); }
+  get path(): string[] {
+    return controlPath(this.name !== null ? this.name.toString() : '', this._parent !);
```

Comment:
I think we want `!=`. Otherwise, `undefined` isn't handled (which is the case in G3). Also, can we flip the ternary so it's not "if not, else" ?

```suggestion
    return controlPath(this.name == null ? '' :  this.name.toString(), this._parent !);
```

Reply:
@kara Done 👍